dotnet/src/dotnetframework/GxClasses/Security/GxSecurityProvider.cs

```diff
@@ -30,7 +30,7 @@ public interface ISecurityProvider
 		GxResult oauthauthentication(IGxContext context, String grantType, String userName, String userPassword, String clientId, String clientSecret, String scope, out OutData outData, out String URL, out bool flag);
 		GxResult oauthauthentication(IGxContext context, String grantType, String userName, String userPassword, String clientId, String clientSecret, String scope, String additionalParameters, out OutData outData, out String URL, out bool flag);
 		void oauthgetuser(IGxContext context, out String userJson, out bool isOK);
-		void oauthlogout(IGxContext context);
+		void oauthlogout(IGxContext context, out String URL, out short statusCode);
```

Comment:
Compatibility issue: cannot change the method signature; we must preserve the older version (defining it as obsolete if it required).